packages/iris-grid/src/AdvancedFilterCreatorSelectValueList.tsx

```diff
@@ -172,28 +208,28 @@ class AdvancedFilterCreatorSelectValueList extends PureComponent {
     return invertSelection ? selectedIndex < 0 : selectedIndex >= 0;
   }
 
-  startListening(table) {
+  startListening(table?: Table): void {
     if (table != null) {
       table.addEventListener(dh.Table.EVENT_UPDATED, this.handleTableUpdate);
     }
   }
 
-  stopListening(table) {
+  stopListening(table?: Table): void {
     if (table != null) {
       table.removeEventListener(dh.Table.EVENT_UPDATED, this.handleTableUpdate);
     }
   }
 
-  resetViewport() {
-    if (this.list) {
+  resetViewport(): void {
+    if (this.list && this.list.topRow && this.list.bottomRow) {
```

Comment:
Check `topRow != null` and `bottomRow != null` instead of the truthy value here. Right now the number `0` will not pass, even though it should.